be/src/olap/rowset/rowset_writer.h

```diff
@@ -62,10 +62,12 @@ class RowsetWriter {
     virtual Status flush_columns() { return Status::Error<ErrorCode::NOT_IMPLEMENTED_ERROR>(); }
     virtual Status final_flush() { return Status::Error<ErrorCode::NOT_IMPLEMENTED_ERROR>(); }
 
-    virtual Status flush_single_memtable(MemTable* memtable, int64_t* flush_size) {
+    virtual Status flush_single_memtable(MemTable* memtable, int64_t* flush_size,
+                                         std::vector<int64_t>* seg_ids_for_delete_bitmap) {
```

Comment:
seg_ids_for_delete_bitmap -> seg_ids may be more generic.

Reply:
ok